discovery-provider/src/queries/notifications.py

```diff
@@ -1073,7 +1073,7 @@ def notifications():
 
 
 def get_max_slot(redis: Redis):
-    listen_milestone_slot = redis.get(latest_sol_listen_count_milestones_slot_key)
+    listen_milestone_slot = redis.get(latest_sol_plays_slot_key)
```

Comment:
is this reliable? i'm also wondering how we can monitor the state of milestones and if the trigger is running behind.